Разметка всех страниц

InnaLady opened the pull request:
Павел, 
отправляю разметку двух файлов

---
:mortar_board: [Разметка всех страниц](https://up.htmlacademy.ru/htmlcss/36/user/2286771/tasks/5)

InnaLady committed:
Update index.html

InnaLady committed:
Update index.html

InnaLady committed:
Update index.html

Добавляет ссылку на страницу Главная

InnaLady committed:
Update index.html

Вешает класс на ссылку

InnaLady committed:
Update index.html

Копирует ссылки

InnaLady committed:
Update index.html

Исправляет ссылку

InnaLady committed:
Update index.html

Заменяет ссылки и вставляет текст из макета

InnaLady committed:
Update index.html

Исправляет ошибки (закрывает теги)
Копирует навигацию

InnaLady committed:
Update index.html

Меняет класс

InnaLady committed:
Update index.html

Меняет ссылки

InnaLady committed:
Update index.html

Меняет текст ссылок

InnaLady committed:
Update index.html

Добавляет класс visually-hidden

InnaLady committed:
Update index.html

Добавляет тег main на страницу

InnaLady committed:
Update index.html

Добавляет класс в теге main

InnaLady committed:
Update index.html

Добавляет заголовок h1 и класс к нему

InnaLady committed:
Update index.html

Добавляет логотип в header
Меняет название заголовка h1

InnaLady committed:
Update index.html

Добавляет логотип в тег main

InnaLady committed:
Update index.html

Изменяет alt в логотипе header и исправляет href на alt в логотипе main-page

InnaLady committed:
Update index.html

Создает второй section и вносит изменения в первый section

InnaLady committed:
Update index.html

Добавляет раздел section с заголовком Заинтересовались?

InnaLady committed:
Update index.html

Добавляет абзац

InnaLady committed:
Update index.html

Добавляет тег form и атрибуты

InnaLady committed:
Update index.html

Добавляет теги  input и button
Исправляет атрибут action у тега form

InnaLady committed:
Update index.html

Добавляет футер, добавляет парные теги

InnaLady committed:
Update index.html

Добавляет класс в footer

InnaLady committed:
Update index.html

Добавляет список

InnaLady committed:
Update index.html

Добавляет список и ссылку

InnaLady committed:
Update index.html

Копирует пункты списка, меняет ссылки

InnaLady committed:
Update index.html

Добавляет телефон, добавляет ссылку на htmlacademy

InnaLady committed:
Create catalog.html

Создает новый файл и создает структуру

InnaLady committed:
Update catalog.html

Меняет загаловок

InnaLady committed:
Update catalog.html

Копирует header из index.html

InnaLady committed:
Update catalog.html

Копирует footer

InnaLady committed:
Update index.html

Вносит правки в footer

InnaLady committed:
Update catalog.html

Добавляет navigation-link-current

InnaLady committed:
Update catalog.html

Добавляет логотип, ведущий на главную страницу

InnaLady committed:
Update catalog.html

Создает тег main с классом main-inner

InnaLady committed:
Update catalog.html

Добавляет header

InnaLady committed:
Update catalog.html

Добавляет тег h1 в тег header

InnaLady committed:
Update catalog.html

Добавляет хлебные крошки

InnaLady committed:
Update catalog.html

Добавляет новый заголовок для секции фильтрации

InnaLady committed:
Update catalog.html

Создает два поля fieldset для фильтров

InnaLady committed:
Update catalog.html

Добавляет первый чекбокс

InnaLady committed:
Update catalog.html

Копирует  чекбоксы

InnaLady committed:
Update catalog.html

Добавляет список из чекбоксов в поле Инфраструктура

InnaLady committed:
Update catalog.html

Добавляет варианты для радиокнопок

InnaLady committed:
Update catalog.html

Вставляет блок select с Сортировкой

InnaLady committed:
Update catalog.html

Добавляет варианты

InnaLady committed:
Update catalog.html

Добавляет список для смены видов карточек

InnaLady committed:
Update catalog.html

Добавляет форму фильтрации и кнопку для отправки данных на сервер

InnaLady committed:
Update catalog.html

Добавляет вторую кнопку сбросить

InnaLady committed:
Update catalog.html

Добавляет абзац Найдено гостиниц и указывает их кол-во

InnaLady committed:
Update catalog.html

Красиво форматирует

InnaLady committed:
Update catalog.html

Добавляет абзац с ценой

InnaLady committed:
Update catalog.html

Добавляет ссылку на ПОДРОБНЕЕ

InnaLady committed:
Update catalog.html

Добавляет ссылку В ИЗБРАНОЕ

InnaLady committed:
Update index.html

Форматируем папку

InnaLady committed:
Update catalog.html

Добавляет тег для декоративных звезд

InnaLady committed:
Update catalog.html

Добавляет тег для рейтинга

InnaLady committed:
Update catalog.html

Меняет тег для декоративного изображения, оборачивает в список и копирует пункты списка

InnaLady committed:
Update catalog.html

Добавляет ссылки г названиям гостиниц, добавляет пагинацию

InnaLady committed:
Update catalog.html

Добавляет пункт списка

InnaLady committed:
Update catalog.html

Добавляет ссылку в пагинацию

InnaLady committed:
Update catalog.html

Копирует пункты списка пагинации  и вносит изменения

InnaLady committed:
Update catalog.html

Добавляет класс к тегу p Найдено гостиниц

InnaLady committed:
Update catalog.html

Убирает лишний второй элемент тега form

InnaLady committed:
Update catalog.html

Исправляет ошибки после валидатора

InnaLady committed:
Update index.html

Исправляет ошибки после проверки на валидаторе

pavel-niukalo commented on index.html:
```diff
@@ -1,12 +1,147 @@
   <body>
+    <header class="page-header">
+      <nav class="navigation">
+        <img class="logo" src="#" alt="Логотип туристического городка Седона">
```
Для единообразия я бы тут тоже ссылку сделал, просто без `href`

Так же не забывай, что изображениям нужно дать размеры (по критерию)

pavel-niukalo commented on index.html:
```diff
@@ -1,12 +1,147 @@
+        <ul class="navigation-user">
+          <li class="navigation-item">
+            <a class="navigation-link" href="#">
+              <span class="visually-hidden">
+                Поиск
+              </span>
```
По критерию скрытому описанию нужно в конце добавлять точку. Относится ко всем таким элементам

![image](https://user-images.githubusercontent.com/52506235/214075370-379b8b97-5be1-43fc-a944-6d00836558a7.png)

![image](https://user-images.githubusercontent.com/52506235/214075404-9ababee8-1396-4c19-83f5-0e207503645b.png)

pavel-niukalo commented on index.html:
```diff
@@ -1,12 +1,147 @@
+    </header>
+    <main class="main-index">
+      <h1 class="visually-hidden">Город Седона</h1>
+      <img class="intro-logo" src="#" alt="Логотип приглашения в туристический городок Седона">
```
Так же про размеры

pavel-niukalo commented on index.html:
```diff
@@ -1,12 +1,147 @@
+          <li class="advantages-item">
+            <h3 class="advantages-title">Настоящий городок</h3>
+            <p class="advantages-description">Седона — не аттракцион для туристов, там течёт своя жизнь</p>
+            <img src="photo-1.png" alt="Домики городка Аризона с горами вдали">
```
Относится ко всем картинкам

pavel-niukalo commented on index.html:
```diff
@@ -1,12 +1,147 @@
+        </div>
+        <ul class="tips-list">
+          <li class="tips-item">
+            <img src="#" alt="Домик">
+            <h3 class="tips-title">Жильё</h3>
+            <p class="tips-description">Рекомендуем пожить в настоящем мотеле, всё как в кино!</p>
+          </li>
+          <li class="tips-item">
+            <img src="#" alt="Бургер">
+            <h3 class="tips-title">Еда</h3>
+            <p class="tips-description">Всегда заказывайте
+              топовый фирменный бургер, вы не разочаруетесь!</p>
+          </li>
+          <li class="tips-item">
+            <img src="#" alt="Подарок">
```
Эти картинки я бы предложил сделать фоновыми, то есть убрать их из разметки и потом добавить через `css`

pavel-niukalo commented on index.html:
```diff
@@ -1,12 +1,147 @@
+        <h2>Заинтересовались?</h2>
+        <p class="search-text">Укажите предполагаемые даты поездки, и мы покажем вам лучшие предложения гостиниц в
+          Седоне</p>
+        <a class="button" href="#">ПОИСК ГОСТИНИЦЫ В СЕДОНЕ</a>
```
Текст можно писать как обычно - "Поиск гостиницы в Седоне". А уже в верхнем регистре сделаем через `css`

pavel-niukalo commented on index.html:
```diff
@@ -1,12 +1,147 @@
+          <input class="text" name="subscribe-e-mail" placeholder="e-mail">
+          <button class="button" type="submit">Подписаться</button>
+        </form>
+      </section>
```
1. Эту секцию я бы добавил в футер, поскольку она есть и в каталоге
2. Нужно добавить скрытый `label` для инпута, по критерию

pavel-niukalo commented on index.html:
```diff
@@ -1,12 +1,147 @@
+          </a>
+        </li>
+      </ul>
+      <p class="telephone">+7 (812) 812-12-12</p>
```
Телефоны должны быть размечены ссылкой с [соответствующим ](https://htmlacademy.ru/blog/html/mailto) `href`

pavel-niukalo commented on catalog.html:
```diff
@@ -0,0 +1,209 @@
+      </header>
+      <section class="catalog-hotels">
+        <h2 class="visually-hidden">Список гостиниц</h2>
+        <form class="catalog-filter" action="https://echo.htmlacademy.ru/" method="get">
+        <fieldset class="catalog-filter-group">
```
Немного поехали отступы, вложенные теги должны отделяться двумя пробелами, то бишь должно быть

```html
<form>
  <fieldset>
```

pavel-niukalo commented on catalog.html:
```diff
@@ -0,0 +1,209 @@
+            <li class="catalog-filter-item">
+              <label class="control">
+                <input class="control-input" type="checkbox" name="Swimming-pool" checked>Бассейн
+              </label>
```
Для удобочитаемости предложил бы подпись писать на новой строке

```html
<label class="control">
  <input class="control-input" type="checkbox" name="Swimming-pool" checked>
  Бассейн
</label>
```

pavel-niukalo commented on catalog.html:
```diff
@@ -0,0 +1,209 @@
+          <option value="expensive">Сначала дорогое</option>
+          <option value="popular">Сначала популярное</option>
+        </select>
+        <a href="#" class="button-cards">
+          <span class="visually-hidden">Показать карточки</span>
+        </a>
+        <a href="#" class="button-gallery">
+          <span class="visually-hidden">Показать галлерею</span>
+        </a>
+        <a href="#" class="button-list">
+          <span class="visually-hidden">Показать список</span>
+        </a>
```
Тут думаю тоже можно списком сделать

pavel-niukalo commented on catalog.html:
```diff
@@ -0,0 +1,209 @@
+          <span class="visually-hidden">Показать список</span>
+        </a>
+        <ul class="hotel-card-list">
+        <li class="hotel-card-item">
+          <span class="visually-hidden">Вид из пещеры на звездное небо</span>
+          <a class="hotel-card-link" href="#">
+          <h3 class="hotel-card-title">Amara Resort & Spa</h3>
+          </a>
+          <p>Гостиница<span>От 4000 ₽</span></p>
+          <a class="hotel-class-details-button button" href="#">ПОДРОБНЕЕ</a>
+          <a class="hotel-class-to-favourite-button button" href="#">В ИЗБРАННОЕ</a>
+          <span class="visually-hidden">Четыре звезды</span>
+          <p class="ranking-button">Рейтинг:<span>8,5</span></p>
+        </li>
```
1. Нужно вставить картинки, поскольку они контентные 
2. И тогда текст спана перейдет в `alt` и нужно будет удалить этот спан
3. Отступы вложенных поправить нужно
4. И текст можно опять же писать по обычным правилам

pavel-niukalo commented on catalog.html:
```diff
@@ -0,0 +1,209 @@
+              <a class="pagination-link">10</a>
+            </li>
+              </ul>
+      </section>
```
Тоже нужно с отступами разобраться

pavel-niukalo commented on catalog.html:
```diff
@@ -0,0 +1,209 @@
+            </li>
+              </ul>
+      </section>
+    </main>
+    <footer class="page-footer">
```
Нет блока с подпиской

pavel-niukalo commented on catalog.html:
```diff
@@ -0,0 +1,209 @@
+          </a>
+        </li>
+      </ul>
+      <p class="telephone">+7 (812) 812-12-12</p>
```
Все замечания из индекса так же относятся и к каталогу

pavel-niukalo commented on catalog.html:
```diff
@@ -0,0 +1,209 @@
+      <p><a class="htmlacademy-link" href="https://htmlacademy.ru/">.html academy</a></p>
+    </footer>
+  </body>
+  </html>
```
Лишние отступы

InnaLady committed:
Update index.html

Добавляет ссылку логотипу и задает размер изображению

InnaLady committed:
Update index.html

ставит точки в конце скрытых описаний

InnaLady committed:
Update index.html

Задает ширину и высоту у изображений, атрибут src у второго логотипа, убирает точки у нескрытых элементов

InnaLady committed:
Update index.html

Задает высоту и ширину у изображений

InnaLady committed:
Update index.html

Убирает картинки из разметки

InnaLady committed:
Update index.html

Убирает верхний регистр

InnaLady committed:
Update index.html

Убирает Раздел subscription в футер, добавляет заголовок ко второй секции Социальные сети и контакты, оборачивает телефон в ссылку, убирает теги <p>

InnaLady committed:
Update index.html

Оборачивает input в label

InnaLady committed:
Update catalog.html

Форматирует

InnaLady committed:
Update catalog.html

Задает ширину и высоту для иконки логотипа туристического городка

InnaLady committed:
Update index.html

Меняет ширину и высота иконки логотипа туристического городка

InnaLady committed:
Update catalog.html

Добавляет точки в конце скрытых подписей

InnaLady committed:
Update catalog.html

Переносит секцию с рассылками в футер

InnaLady committed:
Update catalog.html

Подписи перенесла на новые строки

InnaLady committed:
Update catalog.html

Создает список из карточек с фавиконками

InnaLady committed:
Update catalog.html

Вставляет контентные картинки

InnaLady committed:
Update catalog.html

Меняет регистр у текста на обычный

InnaLady committed:
Update catalog.html

Форматирует

InnaLady committed:
Update index.html

Форматирует

InnaLady committed:
Update catalog.html

Вносит изменения в форму с подпиской

InnaLady committed:
Update index.html

Вносит изменения в форму с подпиской

InnaLady committed:
Update index.html

Меняет type инпута на email по рекомендации валидатора

InnaLady committed:
Update catalog.html

Меняет type в инпуте, меняет вложенность элементов в списке фавиконок по рекомендации валидатора

InnaLady committed:
Update catalog.html

Добавляет в конце подписи alt точки, как в написано в примере задания

InnaLady committed:
Update index.html

Добавляет точки в конце описаний alt как в примере задания

InnaLady committed:
Update catalog.html

Прописывает дополнительные классы у звезд <p class="stars star-4"></p>

InnaLady committed:
Update catalog.html

Форматирует
